### ISSUE TITLE
Keypoint detection inferencing is not working on predefined checkpoints available in centernet_mobilenetv2fpn_512x512_coco17_kpts.tar.gz

### ISSUE THREAD

**Issue by Edwardsleo** [tensorflow/models#10282]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [ ] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [ ] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [ ] I checked to make sure that this issue has not already been filed.

## 1. The entire URL of the file you are using

http://download.tensorflow.org/models/object_detection/tf2/20210210/centernet_mobilenetv2fpn_512x512_coco17_kpts.tar.gz

## 2. Describe the bug

Keypoint detection inferencing is not working on predefined checkpoints available in centernet_mobilenetv2fpn_512x512_coco17_kpts.tar.gz

## 3. Steps to reproduce
Followed the above link
https://github.com/tensorflow/models/blob/master/research/object_detection/colab_tutorials/inference_tf2_colab.ipynb
On **Build a detection model and load pre-trained model weights** section use centernet_mobilenetv2fpn_512x512_coco17_kpts.tar.gz instead of centernet_hg104_512x512_kpts_coco17_tpu

%%bash
wget http://download.tensorflow.org/models/object_detection/tf2/20210210/centernet_mobilenetv2fpn_512x512_coco17_kpts.tar.gz
tar -xf centernet_mobilenetv2fpn_512x512_coco17_kpts.tar.gz
rm centernet_mobilenetv2fpn_512x512_coco17_kpts.tar.gz*

## 4. Expected behavior

Using the pretrained checkpoints available in centernet_mobilenetv2fpn_512x512_coco17_kpts.tar.gz it should detect the keypoints on an image as in https://cocodataset.org/#keypoints-2020

## 5. Additional context

Getting this warning message on inferencing the pretrained checkpoints
WARNING:tensorflow:`input_shape` is undefined or non-square, or `rows` is not in [96, 128, 160, 192, 224]. Weights for input shape (224, 224) will be loaded as the default.

## 6. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- Mobile device name if the issue happens on a mobile device:
- TensorFlow installed from (source or binary):
- TensorFlow version (use command below):
- Python version:
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Comment by jvishnuvardhan** [comment]
@Edwardsleo I could able to run inference with your saved_model and tflite model. Please check the [gist here](https://colab.research.google.com/gist/jvishnuvardhan/2d00d32f6cad799be4a48a49322c5576/object-detection-inference-on-tf-2-and-tf-hub.ipynb). 

Check [this OD tutorial](https://www.tensorflow.org/hub/tutorials/tf2_object_detection) for more details on the inference.
I am not sure about the accuracy of this model but I could able to run inference. From where did you get this model?
